ci/azure/windows.yml

```diff
@@ -18,22 +18,27 @@ jobs:
 
   steps:
     - powershell: |
-        Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
-        Write-Host "##vso[task.prependpath]$HOME/miniconda3/bin"
+        Write-Host "##vso[task.prependpath]$env:CONDA\condabin"
+        Write-Host "##vso[task.setvariable variable=MINICONDA_DIR]$env:CONDA"
       displayName: 'Add conda to PATH'
-    - script: conda update -q -n base conda
-      displayName: Update conda
-    - script: |
-        call activate
-        conda env create -q --file ci\\deps\\azure-windows-$(CONDA_PY).yaml
+    - bash: |
+        source $MINICONDA_DIR/etc/profile.d/conda.sh
+        conda update -q -y -n base conda
+      displayName: 'Update conda'
+    - bash: |
+        source $MINICONDA_DIR/etc/profile.d/conda.sh
```

Comment:
Same question here; I think the task already puts conda on the path so no need to run this script (assuming that's what it does)

Reply:
i found this was necessary too - otherwise conda complains you need to run `conda init`. Let me double check this though.  sourcing conda.sh here seemed to handle everything nicely

Reply:
confirmed I definitely need this otherwise we get:
```
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.
```